docassemble/StudentEvaluations/data/questions/massachusetts_educational_evaluations_request.yml

```diff
@@ -443,6 +446,35 @@ fields:
     none of the above: |
       I don't know.
 ---
+id: special education director information
```

Comment:
I think we removed this question for a specific reason. Why is it being added back?

Reply:
I added this back so that the user could add an email for the special education coordinator. With the email I am able to send an email to the special education coordinator with all of the documents at the end of the interview.